### ISSUE TITLE
Contour chart deploys non-supported Envoy version

### ISSUE THREAD

**Issue by tsaarni** [bitnami/charts#25493]
### Name and Version

bitnami/contour 17.0.7

### What architecture are you using?

amd64

### What steps will reproduce the bug?

Non-supported Envoy version is deployed with Contour

1. Install `helm install my-release oci://registry-1.docker.io/bitnamicharts/contour`
2. observe error `stacktrace for envoy bug` in Envoy logs `kubectl logs daemonsets/my-release-contour-envoy -c envoy`

### Are you using any custom parameters or values?

No

### What is the expected behavior?

Contour 1.28.3 should be deployed with Envoy 1.29.3

### What do you see instead?

Contour 1.28.3 is deployed with Envoy 1.28.2

https://github.com/bitnami/charts/blob/19e939ee53a5ebfb16eef59216d7e413a294540c/bitnami/contour/values.yaml#L650-L652

### Additional information

Following issue was submitted to Contour https://github.com/projectcontour/contour/issues/6385.
The change was introduced by PR https://github.com/bitnami/charts/pull/25117.

**Comment by tsaarni** [comment]
The latest out of Envoy 1.29 branch is 1.29.4 which would work as well ...

A question: how is the version combination of Contour and Envoy determined by the bitnami-bot?

**Comment by carrodher** [comment]
Thanks for letting us know. We have an internal test that should block the release if the Contour-Envoy combo is different from the one specified in the [Compatibility Matrix](https://projectcontour.io/resources/compatibility-matrix/).
This test did not work in the latest release, probably related to a migration of the testing frameworks we use internally.

I just created a task to 1) release a new Contour chart bundling the proper Envoy version and 2) check the test for future occurrences.

**Comment by carrodher** [comment]
A new version was released bundling envoy 1.29.4, see https://github.com/bitnami/charts/pull/25494:
```diff
-    tag: 1.28.2-debian-12-r0
+    tag: 1.29.4-debian-12-r3
```

We'll work on fix the test so we prevent this discrepancy in the future. Thanks again!

**Comment by tsaarni** [comment]
Thank you @carrodher!
